pinot-core/src/main/java/org/apache/pinot/core/startree/v2/builder/MultipleTreesBuilder.java

```diff
@@ -133,4 +133,8 @@ private static SingleTreeBuilder getSingleTreeBuilder(StarTreeV2BuilderConfig bu
       return new OffHeapSingleTreeBuilder(builderConfig, outputDir, segment, metadataProperties);
     }
   }
+
+  public void close() {
```

Comment:
I have the same question as @sunithabeeram on this one. Where is this being called?

Reply:
Hi, @mcvsubbu @sunithabeeram , this is called by `BaseStarTreeV2Test`. Its real purpose is just to call `MultipleTreesBuilder._segment.destroy()`, since could not directly get `_segement` of `MultipleTreesBuilder` so add a new close method...
If `MultipleTreesBuilder._segment` does not destroyed, the `BaseStarTreeV2Test`'s derived tests such as `MaxStarTreeV2Test` will failed in tearDown.
Thank you.

Reply:
Can you post the failure message? What does it complain about?

Reply:
hi, @mcvsubbu , the failure message is:
```
Running org.apache.pinot.core.startree.v2.MaxStarTreeV2Test
SegmentV1V2ToV3FormatConverter - unsupported non-posix filesystem permissions setting
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.701 sec <<< FAILURE! - in org.apache.pinot.core.startree.v2.MaxStarTreeV2Test
testQueries(org.apache.pinot.core.startree.v2.MaxStarTreeV2Test)  Time elapsed: 0.905 sec
tearDown(org.apache.pinot.core.startree.v2.MaxStarTreeV2Test)  Time elapsed: 0.927 sec  <<< FAILURE!
java.io.IOException: Unable to delete file: C:\Users\yu.xue\AppData\Local\Temp\BaseStarTreeV2Test\testSegment\v3\columns.psf
        at org.apache.pinot.core.startree.v2.MaxStarTreeV2Test.tearDown(MaxStarTreeV2Test.java:29)

Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=512m; support was removed in 8.0

Results :

Failed tests:
  MaxStarTreeV2Test.tearDown:29->BaseStarTreeV2Test.tearDown:177 ? IO Unable to ...

Tests run: 2, Failures: 1, Errors: 0, Skipped: 0
```